content/posts/welcome-to-our-beta-github-app.md

```diff
@@ -0,0 +1,72 @@
+
+- **More granular permissions.**
+  The GitHub App allows us to request only the permissions we need
+  -- no more broad, account-wide access.
+- **More control over repository access.**
+  You can choose exactly which repositories Read the Docs can access,
+  instead of granting access to all of them just to import a single one.
+- **No webhooks required.**
+  The GitHub App subscribes to the events it needs automatically when you install it.
+- **No need for deploy keys.**
+  To access private repositories the GitHub App uses a temporary read-only scoped token.
+- **Keeps working, even if access changes.**
+  If the user who originally connected the repository loses access,
+  the integration will continue working, since the GitHub App itself maintains access.
+- **Always in sync.**
+  The GitHub App subscribes to all required events and will always keep your project up to date with your repository.
+
+## Beta access
+
+The new GitHub App already offers the same functionality as our legacy OAuth2 application,
+we are inviting users to try it out and provide feedback,
+so we can smooth out any issues before the full rollout.
+
+Some things you should know before participating in the beta:
+
+- The beta is only available for Read the Docs Community users (<https://app.readthedocs.org>).
+  We will be expanding the beta to Read the Docs Business users (<https://app.readthedocs.com>) in the future.
+- The beta is available for existing and new users.
+- Existing users will be prompted to authorize our new GitHub App.
+  Your existing Read the Docs account will be linked to the new GitHub App,
+  and you won’t lose access to your existing projects.
+- You will be able to migrate your existing projects to the new GitHub App from <https://readthedocs.org/accounts/migrate-to-github-app/>.
+- Your existing projects will continue working using the legacy OAuth2 application,
+  you don't need to migrate them immediately.
+- Once you have connected your account to the new GitHub App,
+  you won't be able to link projects to repositories authorized for the legacy OAuth2 application.
+  You'll need to install the GitHub App into the repository before connecting it to a project on Read the Docs.
+- We recommend that you don't revoke access to the legacy OAuth2 application until the end of the beta period.
+
+If you would like to participate in the beta, you can follow these steps:
+
+- If you are an existing user with your GitHub account already linked to Read the Docs,
+  you can start the migration process [from this page](https://readthedocs.org/accounts/migrate-to-github-app/).
+- If you are a new user, you can sign up using the new GitHub App [from this page](https://readthedocs.org/accounts/githubapp/login/).
+- If you are an existing user, but you don't have your GitHub account linked to Read the Docs,
+  you can link your GitHub account to our new GitHub App [from this page](https://readthedocs.org/accounts/githubapp/login/?process=connect).
```

Comment:
I would change all these `from this page` links to some text more relevant in each case. They all look the same now which is confusing.